node/modules/lp2p/rcmgr.go

```diff
@@ -52,37 +50,41 @@ func ResourceManager(connMgrHi uint) func(lc fx.Lifecycle, repo repo.LockedRepo)
 		// For every extra 1GB of memory we have available, increase our limit by 1GiB
 		defaultLimits.SystemLimitIncrease.Memory = 1 << 30
 		defaultLimitConfig := defaultLimits.AutoScale()
-		if defaultLimitConfig.System.Memory > 4<<30 {
+
+		changes := rcmgr.PartialLimitConfig{}
```

Comment:
The diff in this file looks good to me for the rcmgr type changes. Sorry for the work, but hopefully it removes a foot gun.